starter-code/css/main.css

```diff
@@ -7,4 +7,103 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  height: 1000px;
 }
+
+.logoAndLogin {
```

Comment:
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO, CAMEL CASE PARA LAS CLASES NOOOOOOOOOOOOOOOO, se pone `logo-and-login`